src/forms/Form.jsx

```diff
@@ -56,48 +59,88 @@ const RJSFForm = ({
 
   return (
     <Provider store={store}>
-      <Form
-        className={["__Form__", className].join(" ")}
-        ref={formRef}
-        schema={schema}
-        uiSchema={uiSchema}
-        tagName={tagName}
-        formData={formData}
-        fields={{
-          ...CAPFields,
-          ...customizationContext.customFields,
-          ...fields,
-          ...(isPublished && PublishedFields),
-          ...(isPublished && customizationContext.customPublishedFields),
-        }}
-        widgets={{
-          ...CAPWidgets,
-          ...customizationContext.customWidgets,
-          ...widgets,
-          ...(isPublished && PublishedWidgets),
-          ...(isPublished && customizationContext.customPublishedWidgets),
-        }}
-        templates={templates}
-        liveValidate={liveValidate}
-        showErrorList={showErrorList}
-        noHtml5Validate={true}
-        onError={() => {}}
-        onBlur={() => {}}
-        customValidate={validate}
-        validator={validator}
-        extraErrors={extraErrors}
-        onChange={handleChange}
-        readonly={readonly || isPublished}
-        transformErrors={transformErrors}
-        formContext={{
-          formRef,
-          ...formContext,
-          hideAnchors,
-        }}
-        idSeparator={customizationContext.separator}
+      <ErrorBoundary
+        description={
+          <div style={{ whiteSpace: "normal" }}>
+            <Typography.Text strong>
+              Why am I getting this error?
+            </Typography.Text>
+            <Typography.Paragraph ellipsis={{ rows: 12, expandable: true }}>
+              Your schema might not be following the{" "}
+              <a
+                href="https://json-schema.org/specification"
+                target="_blank"
+                rel="noreferrer"
+              >
+                JSONSchema specification
+              </a>
+              . This usually happens when you have manually modified the JSON
+              schema and introduced some errors. Please make sure the schema
+              follows the specification and try again.
+              <br />
+              Notes:
+              <ul>
+                <li>
+                  Formule adds some custom properties to the schema not present
+                  in the JSONSchema specification, but these should not cause
+                  issues.
+                </li>
+                <li>
+                  When you get this error, you usually want to be looking at
```

Comment:
- since we aim for external use, lets make the error element customizable and passed through the context provider (with a default as you have it).
- in the default we should also update the text to mention that it can also be a uiSchema issue (e.g widget/field non existant, etc)

Reply:
In that case I think we should allow providing a custom ErrorBoundary directly, this way it's even more flexible (you can provide an antd error boundary with a different message or a fully custom one with your own logic, and otherwise we use the default).

Reply:
ok lets go this way